src/pages/Welcome.tsx

```diff
@@ -1,60 +1,72 @@
 import * as React from "react";
+import { Component } from "react";
+import { AppContext } from "../contexts/AppContext";
+import { Project } from "../types";
 import ProjectCard from "../components/ProjectCard";
-import "../style/pages/welcome.css";
+import "../style/pages/Welcome.css";
+import "../style/pages/Projects.css";
 
-function Welcome() {
-  /* TODO: Render ProjectCard components */
-  /* https://github.com/shescoding/projects-platform-frontend/issues/13 */
+const renderProjects = (projects: Array<Project>) => {
```

Comment:
I think this could be exported from Projects.tsx and reused here.

Reply:
Cool, I'll take a look!

Reply:
Ready for re-review in latest commits 😊